src/components/AppProvider/utilities/createPolarisContext/tests/createPolarisContext.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('createPolarisContext()', () => {
       stickyManager: expect.any(StickyManager),
       scrollLockManager: expect.any(ScrollLockManager),
       appBridge: undefined,
-      polarisTheme: {
+      theme: {
```

Comment:
Polaris context will now be under `theme`, only when merging with `AppProvider` context to create `polaris`

Reply:
What do you mean by this?

Reply:
```tsx
const polaris = usePolaris();
console.log(polaris.theme.logo)

const theme = React.useContext(ThemeProviderContext);
theme.logo
```

The `theme` namespace doesn't exist on the theme context, but we'll place all the theme context on a theme key when adding it to app provider context to create `polaris`